Runtime/ConnectionManager.cs

```diff
@@ -108,6 +107,15 @@ public class ConnectionManager {
             Logger.LogError("Invalid bot login, need to re-authenticate");
         }
 
+        private void ClientOnOnJoinedChannel(object sender, OnJoinedChannelArgs e)
+        {

```

Comment:
Just to keep formatting consistent in the code, opening brackets go at the end of the line